layers/+tools/habitrpg/packages.el

```diff
@@ -0,0 +1,29 @@
+;;; packages.el --- habitrpg layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Blasphemy <ksonney@sonnney.com>
+;; URL: https://github.com/ksonney/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `habitrpg/init-PACKAGE' to load and initialize the package.
+
+;;; Code:
+
+(defconst habitrpg-packages
+  '(
+    (habitrpg :location (recipe
+                         :fetcher github
+                         :repo "ryjm/habitrpg.el"))))
```

Comment:
What's the problem with https://github.com/abrochard/emacs-habitica? That package is on MELPA.

Reply:
@TheBB until I can convince the upstream maintainer to  submit the package to MELPA, is that an absolute blocker? As noted above, the linked package does not support org-mode, nor is it as feature complete.